sentry/src/access.rs

```diff
@@ -218,7 +255,7 @@ mod test {
     async fn session_uid_rate_limit() {
         let (config, redis) = setup().await;
 
-        let session = Session {
+        let session = &Session {
```

Comment:
You can simply use `Some(&session)` instead of creating `&Session`

Reply:
This is due to using `session` in multiple places having `Some(&session)` it would be impossible has it would be moved & would require a clone